kubectl-fdb/cmd/root.go

```diff
@@ -145,3 +149,19 @@ func printStatement(cmd *cobra.Command, line string, mesType messageType) {
 	cmd.Printf("✔ %s\n", line)
 	color.Unset()
 }
+
+// will check plugin version and won't let any interaction happen with cluster, if it's not latest release version
+func usingLatestPluginVersion() bool {
+	if latestPluginVersion == "" {
```

Comment:
Can we add an additional flag on the root to skip this check? Just to make sure people can skip this check if they want (default should be that the check is enabled).